### DISCUSSION TITLE
Is it possible to get a scene to the alexa app?

### DISCUSSION THREAD

**Discussion by AriPapari** [Joery-M/Govee-LAN-Control#1] [Q&A]
Hey, still haven't tried out yet but I'm wondering, if it's possible to get configured scenes to work on alexa app. I'd like to use it in some of my alexa routines.

**Comment by Joery-M** [comment]
I personally don't have Amazon Alexa products, so if I make it, I wouldn't be able to test it.
But if someone else wants to help with it, I will gladly add it to the project.
